scripts/build.js

```diff
@@ -46,6 +47,49 @@ function afterBuild(options) {
 }
 
 /**
+ * initSubmodules
+ *
+ * @param {Function} cb
+ * @api private
+ */
+
+function initSubmodules(cb) {
+  var errorMsg = '';
+  var git = spawn(['LIBSASS_GIT_VERSION=', pkg.libsass, ' ./scripts/git.sh'].join());
+  git.stderr.on('data', function(data) {
+    errorMsg += data.toString();
+  });
+  git.on('close', function(code) {
+    var error;
+    if (code !== 0) {
+      error = { message: errorMsg + 'Unable to checkout the libSass submodule' };
+    }
+    cb(error);
+  });
+}
+
+/**
+ * installGitDependencies
+ *
+ * @param {Function} cb
+ * @api private
+ */
+
+function installGitDependencies(cb) {
+  var libsassPath = './src/libsass';
```

Comment:
If we don't have this directory we're almost certainly going to run into issues building node-sass.

If it does exist, which it does when installed via npm (from npm), we do nothing and continue on as before.